README.md

```diff
@@ -54,6 +54,20 @@ NOTE: Please remember that this is a prototype. Its main purpose is to demonstra
 ## Workspace
 Once evo.ninja is run, there will be a `./workspace` directory created. This is the root directory for the agent. Any files will be read & written from this directory.
 
+## Benchmarks
+In order to run Agent Protocol Benchmarks you must have the already mentioned pre-requests and:
+- [python](https://www.python.org/downloads/)
+- [poetry](https://python-poetry.org/docs/#installation)
+
+Then, in one terminal you must start the Agent Protocol HTTP Server: `yarn start:api`; in another terminal you
```

Comment:
Also don't we need some steps for making sure the git submodule is cloned properly?